src/NuGet.Clients/NuGet.CommandLine/NuGetResources.resx

```diff
@@ -6158,4 +6158,13 @@ Oluşturma sırasında NuGet'in paketleri indirmesini önlemek için, Visual Stu
   <data name="ConsolePasswordProvider_PromptForPassword" xml:space="preserve">
     <value>Password: </value>
   </data>
+  <data name="Warning_LongPath_DisabledPolicy" xml:space="preserve">
+    <value>Please enbale long path support in local group policy, more details at https://blogs.msdn.microsoft.com/jeremykuhne/2016/07/30/net-4-6-2-and-long-paths-on-windows-10/.</value>
```

Comment:
Typo: 
Please enable long path support in the local group policy, more details at https://blogs.msdn.microsoft.com/jeremykuhne/2016/07/30/net-4-6-2-and-long-paths-on-windows-10/.

Should we use an aka.ms link because it'll be shorter? /cc @anangaur

Reply:
Thats right. Here is the URL: https://aka.ms/nuget-long-path